presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -731,6 +732,45 @@ public static double chiSquaredCdf(
         return distribution.cumulativeProbability(value);
     }
 
+    @Description("Hypergeometric cdf given populationSize, numberOfSuccesses, sampleSize and observedSuccesses")
+    @ScalarFunction
+    @SqlType(StandardTypes.DOUBLE)
+    public static double hypergeometricCdf(
+            @SqlType(StandardTypes.INTEGER) long populationSize,
+            @SqlType(StandardTypes.INTEGER) long numberOfSuccesses,
+            @SqlType(StandardTypes.INTEGER) long sampleSize,
+            @SqlType(StandardTypes.INTEGER) long observedSuccesses)
```

Comment:
Please change `observedSuccesses` to `value` (just to be consistent with the other functions)